crates/yttrium/src/account_client.rs

```diff
@@ -131,8 +140,67 @@ impl AccountClient {
         .await
     }
 
-    pub async fn sign_message(&self, message: String) -> eyre::Result<String> {
-        todo!("Implement sign_message: {}", message)
+    pub fn prepare_sign_message(
```

Comment:
Kotlin, It would copy these methods to account_client in kotlin-ffi. With some tweaks, it should work